### PR TITLE
16.0

### PR DESCRIPTION
**Pull request by pq**
@devoncarew @stevemessick

### CODE REVIEW

File: resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -22,6 +22,39 @@
   <change-notes>
     <![CDATA[
 
+16.0:
+<ul>
+  <li>device list refresh fixes</li>
+  <li>support for flutter run in profile and release modes</li>
+  <li>support to pull the android sdk location from flutter tools</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: 'to pull' => 'for reading'

File: resources/META-INF/plugin.xml [hunk 2]
```diff
@@ -22,6 +22,39 @@
...
+  <li>support for discovering and running Flutter widget tests</li>
+  <li>Flutter test console improvements</li>
+  <li>support for running flutter doctor in a Bazel workspace</li>
+  <li>test file icon annotations</li>
+  <li>support to locate a missing flutter SDK in .packages files</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: for locating

File: resources/META-INF/plugin.xml [hunk 3]
```diff
@@ -22,6 +22,39 @@
...
+  <li>open emulator action sorting</li>
+  <li>test state icons for Flutter tests</li>
+  <li>editor line markers for Flutter tests</li>
+  <li>new restart daemon action</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: added a new

File: resources/META-INF/plugin.xml [hunk 4]
```diff
@@ -22,6 +22,39 @@
...
+  <li>open emulator action sorting</li>
+  <li>run/debug button enablement improvements</li>
+  <li>fix to ensure `Install SDK…` action is always visible</li>
```
Comment (devoncarew) [inline resources/META-INF/plugin.xml]: ensure the

**Comment by devoncarew** [comment]
lgtm

File: resources/META-INF/plugin.xml (hunk 1 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: Done!

File: resources/META-INF/plugin.xml (hunk 2 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: 👍

File: resources/META-INF/plugin.xml (hunk 3 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: 🎉

File: resources/META-INF/plugin.xml (hunk 4 above)
Comment (pq) [inline resources/META-INF/plugin.xml]: Done.

Thanks for the wordsmithing!

**Pull request merged**